commandGroups/ops.json

```diff
@@ -0,0 +1,1963 @@
+{
```

Comment:
I don't think we have been explicitly told we can do this yet.

Reply:
removed